WooCommerce/Classes/Tools/InfiniteScroll/PaginationTracker.swift

```diff
@@ -61,16 +61,18 @@ final class PaginationTracker {
     ///     1.  Proceed only if a given element is the last one in it's page
     ///     2.  Verify if the nextpage isn't currently being synced
     ///     3.  Proceed syncing the next page, if possible / needed
-    func ensureNextPageIsSynced() {
+    func ensureNextPageIsSynced(onCompletion: (() -> Void)? = nil) {
```

Comment:
nit: `PaginationTracker` diffs can be reverted now that we have `AsyncPaginationTracker`.

Reply:
Reverted in https://github.com/woocommerce/woocommerce-ios/pull/14727/commits/1d3ffd28d215891fd7b58663ffe718b062bbbe09.